<launch>

  <!-- joy node -->
  <node respawn="true" pkg="joy" type="joy" name="joystick" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

  <node pkg="joystick_ps3" type="joystick_publisher" name="JoystickPublisher"/>
</launch>
